fybrik/README_Chaining.md

```diff
@@ -0,0 +1,62 @@
+of the dataset columns must be redacted. The airbyte module currently has no data-transformation capabilities.
+Therefore, to satisfy the constraints, Fybrik must deploy two modules: the airbyte module for reading the
+dataset, and the [arrow-flight-module](https://github.com/fybrik/arrow-flight-module) for transforming the
+dataset based on the governance policies.
+
+The use case we describe is similar to the use case out like [here](fybrik/README.md). The difference is that
+in this use case the governance policies mandate a data transformation. Therefore, the main difference is in the
+rego policy file: [sample-policy-restrictive.rego](sample-policy-restrictive.rego) rather than [sample-policy.rego](sample-policy.rego).
+
+We explain how, using an Airbyte FybrikModule, a workload can access data stored in google-sheets, postgres, and other data stores supported by Airbyte connectors. To do so a FybrikApplication (i.e. the request) must be submitted indicating the desired data set(s). In this example, we use the `userdata` dataset, a Parquet file found in https://github.com/Teradata/kylo/blob/master/samples/sample-data/parquet/userdata2.parquet.
+
+You will need a copy of the Fybrik repository (`git clone https://github.com/fybrik/fybrik.git`). Set the following environment variables: FYBRIK_DIR for the path of the `fybrik` directory, and AIRBYTE_MODULE_DIR for the path of the `airbyte-module` directory.
+
+1. Install Fybrik Prerequisites. Follow the instruction in the Fybrik [Quick Start Guide](https://fybrik.io/dev/get-started/quickstart/). Stop before the "Install control plane" section.
+
+1. Before installing the control plane, we need to customize the [Fybrik taxonomy](https://fybrik.io/dev/tasks/custom-taxonomy/), to define new connection and interface types. Run:
+    ```bash
+    cd $FYBRIK_DIR
+    go run main.go taxonomy compile --out custom-taxonomy.json --base charts/fybrik/files/taxonomy/taxonomy.json $AIRBYTE_MODULE_DIR/fybrik/fybrik-taxonomy-customize.yaml
+    helm install fybrik-crd charts/fybrik-crd -n fybrik-system --wait
+    helm install fybrik charts/fybrik --set global.tag=master --set global.imagePullPolicy=Always -n fybrik-system --wait --set-file taxonomyOverride=custom-taxonomy.json
+    ```
+
+1. Install the Airbyte module:
+    ```bash
+    kubectl apply -f $AIRBYTE_MODULE_DIR/module.yaml -n fybrik-system
+    ```
+
+1. In this scenaio, we also need the arrow-flight module:
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/fybrik/arrow-flight-module/master/module.yaml -n fybrik-system
+    ```
+
+1. Create a new namespace for the application, and set it as default:
+   ```bash
+   kubectl create namespace fybrik-airbyte-sample
+   kubectl config set-context --current --namespace=fybrik-airbyte-sample
+   ```
+
+1. Create an asset (the `userdata` asset), the policy to access it (we use a policy that requires redactions to PII columns), and an application that requires this asset:
```

Comment:
Create an asset in fybrik's mini data catalog, .... an a FybrikApplication indicating the workload, context, and data requested.